### ISSUE TITLE
Improve reddit search query to get better results

### ISSUE THREAD

**Issue by apoorvapendse** [apoorvapendse/CruSyt#5]
### Edit the query text using regex or any other method to fetch better results

**Comment by arjun-gr** [comment]
Hey apoorva can you please explain the topic more? If i am not wrong are you trying to get the results based on search query? regardless of the top 10 comments?

**Comment by apoorvapendse** [comment]
@arjun-gr  Yeah right now, the implementation is just to scrape the comments from reddit, but i would like them to be more accurate and pin pointed towards posts that actually review a product instead of randomly fetching top 10 comments containing the search query
